src/test/kotlin/uk/gov/dluhc/printapi/rest/GenerateTemporaryCertificateIntegrationTest.kt

```diff
@@ -63,8 +81,76 @@ internal class GenerateTemporaryCertificateIntegrationTest : IntegrationTest() {
                 )
             )
             .contentType(MediaType.APPLICATION_JSON)
+            .body(
+                Mono.just(buildGenerateTemporaryCertificateRequest()),
+                GenerateTemporaryCertificateRequest::class.java
+            )
             .exchange()
             .expectStatus()
             .isForbidden
     }
+
+    @Test
```

Comment:
These 2 integration tests are not testing the validation rules per se, but are about testing that the request handling has invoked both the standard JSR-303 validation, plus our custom initBinder validation

In respect of the initBinder test, one could argue that its testing one of the scenarios that is also covered in the unit test. Whilst this is true, the objective is not about unit testing the validation class/rules, and is more about testing the validator is bound to the request and is invoked.